assets/javascripts/discourse/lib/activity-pub-utilities.js

```diff
@@ -0,0 +1,9 @@
+export function buildHandle({ actor, model, site }) {
+  if ((!actor && !model) || (model && !site)) {
+    return undefined;
+  } else {
+    const username = actor ? actor.username : model.activity_pub_username;
+    const domain = actor ? actor.domain : site.activity_pub_host;
+    return `${username}@${domain}`;
```

Comment:
```suggestion
    return `@${username}@${domain}`;
```

Is this a better standard for ActivityPub handles? Maybe it helps with distinguishing AP handles from emails as well, in places like the follower list, for example.

Reply:
Yes, I considered that. The slight issue I had with it is that there seems to be some conceptual confusion in Mastodon as to what exactly the leading `@` symbol is doing. For example, it seems to be partly there as "mention" concept (see e.g. https://github.com/mastodon/mastodon/issues/16488#issuecomment-877792646). However, it's not used in their webfinger implementation (https://docs.joinmastodon.org/spec/webfinger/). So, the "canonical" handle is without the `@` prefix. That said, I see that Discourse now also adds the `@` symbol prior to the username on the user card. So maybe that makes sense for display purposes here too. Ok, I'm convinced, I'll add it.